scripts/create-test-category.js

```
@@ -0,0 +1,234 @@
+
+    // Login with the test user
+    const response = await api.post('/api/auth/login', {
+      username: TEST_USERNAME,
+      password: TEST_PASSWORD
+    }, {
+      headers: {
+        'X-CSRF-Token': csrfToken
+      }
+    });
+
+    if (response.data && response.data.token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)